src/keylessBackup/KeylessBackupIntro.tsx

```diff
@@ -1,69 +1,97 @@
 import SmartphoneIcon from 'src/keylessBackup/SmartphoneIcon'
 import { KeylessBackupFlow } from 'src/keylessBackup/types'
-import { headerWithBackButton } from 'src/navigator/Headers'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import { default as Colors, default as colors } from 'src/styles/colors'
+import { StackParamList } from 'src/navigator/types'
+import { Colors } from 'src/styles/colors'
 import { typeScale } from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
-
-function onPressContinue() {
-  ValoraAnalytics.track(KeylessBackupEvents.cab_setup_continue)
-  navigate(Screens.SignInWithEmail, { keylessBackupFlow: KeylessBackupFlow.Setup })
-}
-
 function onPressRecoveryPhrase() {
   ValoraAnalytics.track(KeylessBackupEvents.cab_setup_recovery_phrase)
   navigate(Screens.BackupIntroduction)
 }
 
-function SetUpKeylessBackup() {
+type Props = NativeStackScreenProps<StackParamList, Screens.KeylessBackupIntro>
+
+function KeylessBackupIntro({ route }: Props) {
+  const { keylessBackupFlow } = route.params
+  const isSetup = keylessBackupFlow === KeylessBackupFlow.Setup
   const { t } = useTranslation()
+
   return (
     <SafeAreaView style={styles.container}>
+      <CustomHeader left={isSetup ? <BackButton /> : <CancelButton />} style={styles.header} />
       <ScrollView style={styles.scrollContainer}>
-        <Text style={styles.title}>{t('setUpKeylessBackup.title')}</Text>
-        <Text style={styles.subtitle}>{t('setUpKeylessBackup.subtitle')}</Text>
+        {isSetup && <Text style={styles.title}>{t('keylessBackupIntro.setup.title')}</Text>}
+        <Text
+          style={[
+            isSetup ? { ...typeScale.bodyMedium } : { ...typeScale.labelMedium },
+            styles.description,
+          ]}
+        >
+          {isSetup
+            ? t('keylessBackupIntro.setup.description')
+            : t('keylessBackupIntro.restore.description')}
+        </Text>
         <Card style={styles.authFactorsCard} shadow={null}>
           <View style={styles.authFactorsContainer}>
             <View style={styles.authFactorLine}>
               <EnvelopeIcon style={styles.envelopeIcon} />
-              <Text style={styles.authFactorText}>{t('setUpKeylessBackup.emailAddress')}</Text>
+              <Text style={styles.authFactorText}>{t('keylessBackupIntro.emailAddress')}</Text>
             </View>
             <View style={styles.authFactorLine}>
               <SmartphoneIcon style={styles.smartphoneIcon} />
-              <Text style={styles.authFactorText}>{t('setUpKeylessBackup.phoneNumber')}</Text>
+              <Text style={styles.authFactorText}>{t('keylessBackupIntro.phoneNumber')}</Text>
             </View>
           </View>
 
           <Text style={styles.reminderText}>
-            <Trans i18nKey="setupKeylessBackupReminder">
-              {/* namespaced keys did not work for this */}
```

Comment:
interesting that this works now

Reply:
I think what made the difference was using curlies e.g. `i18nKey={'keylessBackupIntro.setup.reminder'}`.